Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -1590,6 +1592,102 @@ public void ToggleMute(ProductUserId targetUserId, OnLobbyCallback ToggleMuteCom
             }
         }
 
+        /// <summary>
+        /// Mute a Member in the Lobby. This only affects the Local Member, other remote Members in the Lobby will still be able to hear them.
+        /// </summary>
+        /// <param name="isMicOn">Active status of the mic</param>
+        /// <param name="targetProductUserId">The Product User ID of the target member to mute</param>
+        /// <param name="MuteMemberCompleted">Callback when MuteMember is completed</param>
+        public void MuteMember(bool isMicOn, ProductUserId targetProductUserId, OnLobbyCallback MuteMemberCompleted)
```

Comment:
I feel like the existing ToggleMute should be changed to determine the target status of the mute operation, and then call this function. I think this repeats enough of the logic that we should certainly not have both.